doc/config.rst

```diff
@@ -1959,6 +1959,20 @@ These options influence text output.
 
    .. versionadded:: 1.1
 
+.. confval:: text_add_secnumbers
+
+   A boolean that decides whether section numbers are included in text output.
+   Default is ``False``.
```

Comment:
I think it is better to enabled this by default. I prefer to provide better default settings instead of compatibility. What do you think?

Reply:
I have no strong opinion either way.

Reply:
Okay, let's enable this by default!